terraform-plans/configs/charm-sysconfig_main.tfvars

```diff
@@ -18,8 +18,8 @@ templates = {
     vars = {
       # Skip ARM64 check because the functional test runs on lxd VM which is not working
```

Comment:
I guess now we can remove this comment because we are adding arm64?

Reply:
Removed